README.md

```diff
@@ -1,600 +1,60 @@
-# Salesforce Ballerina Connector
+# Ballerina Salesforce Connector
 [![Build](https://github.com/ballerina-platform/module-ballerinax-sfdc/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-sfdc/actions?query=workflow%3ACI)
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-sfdc.svg)](https://github.com/ballerina-platformmodule-ballerinax-sfdc/commits/master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Connects to Salesforce from Ballerina.
+# Salesforce Data APIs
 
-# Introduction
-## Salesforce Data APIs
+Salesforce has a vast landscape of APIs since they follow an API-first approach to building features on the Salesforce Platform. This approach gives users flexibility to manipulate their data however they want. The most commonly used Salesforce Data APIs are REST API, SOAP API, Bulk API and Streaming API and together they make up the Salesforce Data API. 
 
-Salesforce has a vast landscape of APIs since they follow an API-first approach to building features on the Salesforce 
-Platform. This approach gives users flexibility to manipulate their data however they want. The most commonly used 
-Salesforce Data APIs are REST API, SOAP API, Bulk API and Streaming API and together they make up the Salesforce Data 
-API. 
+Ballerina Salesforce connector currently utilizes the Salesforce REST API and Bulk API for convenient data manipulation. The Salesforce connector allows users to perform CRUD operations for SObjects, query using SOQL, search using SOSL, and describe SObjects and organizational data through the Salesforce REST API. Also, it supports adding bulk data jobs and batches of types JSON, XML, and CSV via the Salesforce Bulk API. Apart from these functionalities Ballerina Salesforce Connector includes a listener module to capture events.
+For more information about configuration and operations, go to the module.
+- [ballerinax/sfdc](https://docs.central.ballerina.io/ballerinax/sfdc/3.0.0)
+- [ballerinax/sfdc.rest](https://docs.central.ballerina.io/ballerinax/googleapis.sfdc.rest/3.0.0)
+- [ballerinax/sfdc.soap](https://docs.central.ballerina.io/ballerinax/googleapis.sfdc.soap/3.0.0)
```

Comment:
Don't provide versions in the link
Please change the links to the latest agreed format

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sfdc/commit/fe3b44e5034590ee895b9a94623e56e286203103